packages/opentelemetry-web/src/StackContextManager.ts

```diff
@@ -112,7 +112,7 @@ export class StackContextManager implements ContextManager {
     this._currentContext = context || Context.ROOT_CONTEXT;
 
     try {
-      return fn.apply(context);
+      return fn();
```

Comment:
@obecny would like to get your reviews on this.

Reply:
hmm it was a bit tricky from what I remember, but if the all tests are working fine, then I guess it should be fine too ?. Can you check the the examples replacing the zone scope manager with the stack if nothing is missing ?.